Koch Curve/koch curve.py

```diff
@@ -0,0 +1,32 @@
+#importing the libraries
+#turtle standard graphics library for python
+from turtle import *

```

Comment:
since your importing turtle, please mention in the requirements,txt which version you are using

Reply:
yeah sure.

Reply:
turtle is an inbuilt module in python so should i only mention the python version or something else?

Reply:
no need to add now. I will check it later.
Merging this